topics/statistics/metadata.yaml

```diff
@@ -0,0 +1,60 @@
+---
+name: "statistics"
+type: "use"
+title: "Statistics"
+summary: "Statistical Analyses"
+docker_image: "quay.io/galaxy/statistics-training"
+
+requirements:
+  -
+    title: "Galaxy introduction"
+    type: "internal"
+    link: "/introduction/"
+
+material:
+  -
+    title: "Interval-Wise Testing for omics data"
+    type: "tutorial"
+    name: "iwtomics"
+    zenodo_link: "https://doi.org/10.5281/zenodo.1184682"
+    galaxy_tour: yes
+    hands_on: yes
+    slides: no
+    workflows: no
```

Comment:
Do you plan to add the workflow? It would be nice. We are adding automatic tests of the tutorials using the workflows

Reply:
I made a workflow for this tool but I did never test it
[https://test.galaxyproject.org/u/fabiocumbo/w/iwtomics-workflow](https://test.galaxyproject.org/u/fabiocumbo/w/iwtomics-workflow)

Let me make some tests first

Reply:
Tests, as in planemo test workflows? :)

Reply:
it works @bgruening :smiley:
[https://usegalaxy.org/u/fabio-cumbo/w/iwtomics-workflow-training-materials](https://usegalaxy.org/u/fabio-cumbo/w/iwtomics-workflow-training-materials)

Reply:
You mean you tested this with planemo or manually?

Reply:
oh wait.. it does not work actually.. let me make some more test

Reply:
ok, it works now, tested manually